gvr-jassimp/app/src/main/java/org/gearvrf/gvrjassimp/JassimpModelLoaderMain.java

```diff
@@ -67,28 +94,26 @@ public void onInit(GVRContext gvrContext) {
             astroBoyModel.getTransform().setRotationByAxis(45.0f, 0.0f, 1.0f, 0.0f);
             astroBoyModel.getTransform().setScale(3, 3, 3);
             astroBoyModel.getTransform().setPosition(0.0f, -0.4f, -0.5f);
-
-            mMainScene.addSceneObject(astroBoyModel);
         } catch (IOException e) {
             Log.e(TAG, "Failed to load a model: %s", e);
         }
 
         // Model with color
         try {
-            GVRSceneObject benchModel = gvrContext.loadModel("bench.dae");
+            GVRSceneObject benchModel = gvrContext.getAssetLoader().loadModel("bench.dae", mMainScene);
 
             benchModel.getTransform().setScale(0.66f, 0.66f, 0.66f);
             benchModel.getTransform().setPosition(0.0f, -4.0f, -20.0f);
             benchModel.getTransform().setRotationByAxis(180.0f, 0.0f, 1.0f, 0.0f);
-            mMainScene.addSceneObject(benchModel);
             } catch (IOException e) {
             Log.e(TAG, "Failed to load a model: %s", e);
         }
 
         // Model over network
         String urlBase = "https://raw.githubusercontent.com/gearvrf/GearVRf-Demos/master/gvrjassimpmodelloader/assets/";
```

Comment:
The url needs to be updated to https://github.com/gearvrf/GearVRf-Demos/raw/master/gvr-jassimp/app/src/main/assets/